src/events.cpp

```diff
@@ -108,6 +108,14 @@ bool Events::load()
 				info.playerOnWrapItem = event;
 			} else if (methodName == "onInventoryUpdate") {
 				info.playerOnInventoryUpdate = event;
+			} else if (methodName == "onStorageUpdate") {
+				info.playerOnStorageUpdate = event;
+			} else if (methodName == "onQuestTracker") {
+				info.playerOnQuestTracker = event;
+			} else if (methodName == "onQuestLog") {
+				info.playerOnQuestLog = event;
+			} else if (methodName == "onQuestLine") {
+				info.playerOnQuestLine = event;
```

Comment:
I think these can be removed if you move the packet parsing to Lua :)

Reply:
yeah <3